src/System.Collections.Immutable/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
@weshaggard ignore removing this one. I'll try to get this back as I am ifdef teh code